lacework/schema_validators.go

```diff
@@ -41,3 +42,22 @@ func StringDoesNotHavePrefix(chars string) schema.SchemaValidateDiagFunc {
 		return warnings, errors
 	})
 }
+
+// ValidateTimeFormat returns a SchemaValidateFunc which validates that the
+// value is in the timeformat supplied.
+func ValidateTimeFormat(timeFormat string) schema.SchemaValidateDiagFunc {
+	return validation.ToDiagFunc(func(i interface{}, k string) (warnings []string, errors []error) {
+		v, ok := i.(string)
+		if !ok {
+			errors = append(errors, fmt.Errorf("expected type of %s to be string", k))
+			return warnings, errors
```

Comment:
use naked returns instead
```suggestion
			return
```